src/Uno.UI.Runtime.Skia.AppleUIKit/UI/Xaml/RootViewController.cs

```diff
@@ -67,26 +67,40 @@ public RootViewController(NativeHandle handle)
 
 	public void Initialize()
 	{
+		var view = View!;
+
+		_inputLayer = new InputLayer();
+		_inputLayer.Frame = view.Bounds;
+		_inputLayer.AutoresizingMask = UIViewAutoresizing.All;
+		// We inject the input layer directly as `View` in order to get all pointer events, including under the nativeOverlayLayer (unless handled by native view)
+		// Note: We must have a layer (compared to override TouchesXXX methods in this class) in order to be able to properly get the multitouch events.
+		View = view = _inputLayer;
+
 		_textInputLayer = new UIView();
+		view.AddSubview(_textInputLayer);
+
 		_skCanvasView = new SkiaCanvas();
 #if !__TVOS__
 		_skCanvasView.SetOwner(this);
 #endif
-		_skCanvasView.Frame = View!.Bounds;
+		_skCanvasView.Frame = view.Bounds;
 		_skCanvasView.AutoresizingMask = UIViewAutoresizing.All;
 #if __TVOS__
 		_skCanvasView.PaintSurface += OnPaintSurface;
 #endif
-		View!.AddSubview(_textInputLayer);
-		View!.AddSubview(_skCanvasView);
+		view.AddSubview(_skCanvasView);
+
+		_inputLayer = new InputLayer();
+		_inputLayer.Frame = view.Bounds;
+		_inputLayer.AutoresizingMask = UIViewAutoresizing.All;
+		view.AddSubview(_inputLayer);
```

Comment:
Confused with this line here. I see you're adding another `InputLayer` view as a child of the main view, which is already an `InputLayer`

```code
RootViewController
 InputLayer
  -> TextInputLayer
  -> SkCanvasView
  -> InputLayer // => What will be the purpose of this other InputLayer?
  -> NativeOverlayLayer
```

Reply:
Hummm good catch ... it's just the trace of all my many tests (and hopefully this double input layers was not the source of my "success" :/)